projects/solar/src/solar_config.c

```diff
@@ -19,28 +19,21 @@
 #define DATA_TX_MSGS_PER_TX_ITERATION 8
 
 // Scaling factor to convert MCP3427 ADC values (LSB = 62.5uV) for voltage sense to millivolts.
-// Experimentally determined to be ~1.184mV/LSB.
-#define SOLAR_MCP3427_VOLTAGE_SENSE_SCALING_FACTOR 1.184f
+// Must be calibrated.
+#define SOLAR_MCP3427_VOLTAGE_SENSE_SCALING_FACTOR 1.0f
 
 // Scaling factor from MCP3427 ADC values for current sense to microamps.
 // MCP3427 uses 62.5uV/LSB, ACS722LLCTR-10AU has sensitivity of 264mV/A = 0.264uV/uA.
-// (62.5uV/LSB)/(0.264uV/uA) = 236.74uA/LSB. Experimental value is 228.25uA/LSB, which is close.
-#define SOLAR_MCP3427_CURRENT_SENSE_SCALING_FACTOR 228.25f
-
-// Bias of MCP3427 ADC for current sense, in microamps.
-// ACS722LLCTR-10AU gives a bias of 0.33V, corresponding to 1.25A.
-// The experimental value of 1.2527A is close to this.
-#define SOLAR_MCP3427_CURRENT_SENSE_BIAS 1.2527e6f
+// (62.5uV/LSB)/(0.264uV/uA) = 236.74uA/LSB
+#define SOLAR_MCP3427_CURRENT_SENSE_SCALING_FACTOR 236.74f
 
 // Scaling factor to convert SPV1020 current values from SPI to microamps.
-// This is based on ONE data point: reading 46 for a current of 600-800mA, so call it 700mA.
-// 700mA/46LSB ~ 15220uA/LSB.
-// TODO(SOFT-282): get more data points to calibrate this.
-#define SOLAR_MPPT_CURRENT_SCALING_FACTOR 15220.0f
+// Must be calibrated.
+#define SOLAR_MPPT_CURRENT_SCALING_FACTOR 1.0f
 
-// Scaling factor to convert SPV1020 input voltage values from SPI to millivolts.
-// Experimentally determined to be ~26.1mV/LSB.
-#define SOLAR_MPPT_VIN_SCALING_FACTOR 26.1f
+// Scaling factor to convert SPV1020 VIN values from SPI to millivolts.
+// Must be calibrated.
+#define SOLAR_MPPT_VIN_SCALING_FACTOR 1.0f
```

Comment:
I think this was a merge conflict resolution, but please put this back to what it is on master, there's a lot of experimentally-determined parameters in there